esp-hal/src/gpio/mod.rs

```diff
@@ -307,7 +308,7 @@ pub trait RtcPinWithResistors: RtcPin {
 }
 
 /// Common trait implemented by pins
-pub trait Pin: Sealed {
+pub trait Pin: Sealed + Unpin {
```

Comment:
Why is this specifically required? Is this something we need to be concerned about with other driver traits?

Reply:
Umm I think I just forgot to remove this, I think I undid the relevant change in the Future that needed it